Sources/App/Features/Push/Controllers/PushController.swift

```diff
@@ -43,4 +44,25 @@ struct PushController: RouteCollection {
 
         return .noContent
     }
+
+    private func sendNotification(req: Request) async throws -> HTTPStatus {
+        struct SendNotificationRequest: Decodable {
+            let message: String
+            let securityCode: String
+        }
+
+        let notificationRequest = try req.content.decode(SendNotificationRequest.self)
+        let secretSecurityCode = Environment.get("PUSH_SECURITY_CODE")
```

Comment:
nit: Would be nice to wrap this in a guard and provide a different (more helpful) error message if the security code hasn't been set. We should also ensure the code isn't empty.

Reply:
This was just a temporary solution until we can have a mechanism built into the admin area, so the option is there if required.